python/test_infra/test_ttir_ops.py

```diff
@@ -0,0 +1,221 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# RUN: %python %s
+
+import inspect
+
+from ttmlir.test_utils import compile_and_convert
+from ttmlir.ttir_builder import Operand, TTIRBuilder
+
+
+@compile_and_convert(((128, 128),))
```

Comment:
Singleton tuples are a gotcha that I'd prefer to avoid, perhaps we should just make this a list to avoid them?  They always result in a weird error message that is totally unrelated to the fix.

Reply:
Also, sorry just noticed, this should be on the other PR (#1236) . These changes won't appear once that one is merged. This applies to both comments

Reply:
No worries, we can land your other PR first, I accepted that one